components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint/src/main/webapp/self-registration-process.jsp

```diff
@@ -138,6 +139,16 @@
             try {
 
                 for (Claim claim : claims) {
+
+                    if(claim.getUri().trim().contains("/claims/locality") &&
```

Comment:
Better to move this `if` condition as a `else if` condision in https://github.com/wso2/carbon-identity-framework/pull/690/files#diff-df61c586fa1b23706489ed8be0a2fa3eL146

So it would like,
```
if (StringUtils.isNotBlank(request.getParameter(claim.getUri()))) {
   // add normal claim
} else if (claim.getUri().trim().equals("http://wso2.org/claims/locality") && StringUtils.isNotBlank(userLocale)) {
   // add locale claim
}
```